cookbooks/clearwater/files/default/cacti/templates/cacti_host_template_bono.xml

```diff
@@ -0,0 +1,1856 @@
+<cacti>	
```

Comment:
How do we keep these templates in sync with those referenced by http://clearwater.readthedocs.org/en/latest/Cacti/?